### DISCUSSION TITLE
change default name (use-color-scheme)

### DISCUSSION THREAD

**Discussion by masb0ymas** [mantinedev/mantine#4873] [Feedback]
In the future can I change the default name for the `use-color-scheme` setting, currently the default name is `mantine-color-scheme`

```js
const { setColorScheme } = useMantineColorScheme();
```

Future : 
```js
const { setColorScheme } = useMantineColorScheme('my-color-scheme');
```

**Comment by rtivital** [comment]
What does the parameter mean? What does it change?

**Reply by masb0ymas** [reply]
I thought I could set parameters in the hooks function, but it turns out I can set them in MantineProvider, it's cool. Turns out I wasn't the one reading the documentation carefully. Thank you for responding to my response

```js
import { MantineProvider, localStorageColorSchemeManager } from '@mantine/core';

const colorSchemeManager = localStorageColorSchemeManager({ key: 'my-app-color-scheme' });

function Demo() {
  return (
    <MantineProvider colorSchemeManager={colorSchemeManager}>
      <App />
    </MantineProvider>
  );
}
```
